pkg/assembler/backends/neo4j/pkg.go

```diff
@@ -315,7 +315,7 @@ func (c *neo4jClient) Packages(ctx context.Context, pkgSpec *model.PkgSpec) ([]*
 				queryValues["pkgSubpath"] = pkgSpec.Subpath
 			}
 
-			if pkgSpec.MatchOnlyEmptyQualifiers != nil && !*pkgSpec.MatchOnlyEmptyQualifiers {
+			if !*pkgSpec.MatchOnlyEmptyQualifiers {
```

Comment:
ah ok so we assume anything with a ! will definitely be non nil yea

Reply:
In this case the `MatchOnlyEmptyQualifiers` defaults to:
```
 matchOnlyEmptyQualifiers: Boolean = false
 ```
 at the schema level. So it always default to `false`

Reply:
👍